pkg/volume/azure_dd/azure_common.go

```diff
@@ -39,7 +39,7 @@ import (
 
 const (
 	defaultStorageAccountType       = compute.StandardLRS
-	defaultAzureDiskKind            = v1.AzureManagedDisk
+	defaultAzureDiskKind            = v1.AzureSharedBlobDisk
```

Comment:
The reason is ACS engine is not setting kind value for Blob case in file kubernetesmasteraddons-unmanaged-azure-storage-classes.yaml.
Because of which Azure_provision(provision function) always setting the default value to Managed disk.
This is not case for Managed disk case. As the Yaml file has the Kind value passed.

I thought about it and I think it will be good to update ACSEngine to pass the value and revert this change back to Managed Disk.